src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -302,6 +302,13 @@ internal static class Defaults
 
         PredictionKind ITrainer.PredictionKind => PredictionKind.Recommendation;
 
+        /// <summary>
+        /// The <see cref="TrainerInfo"/> contains general parameters for this trainer.
+        /// </summary>
+        TrainerInfo ITrainer.Info => _info;
```

Comment:
You can remove this. It is no longer needed now that we expose the `public TrainerInfo Info`.

Reply:
But put the XML doc comment above on the public property below.

---
In reply to: [335631831](https://github.com/dotnet/machinelearning/pull/4246#discussion_r335631831) [](ancestors = 335631831)